pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -676,3 +682,21 @@ message RemoveImageRequest {
 }
 
 message RemoveImageResponse {}
+
+enum ConfigKey {
+    POD_CIDR = 1;
+    HAIRPIN_MODE = 2;
+    NON_MASQUERADE_CIDRS = 3;
+    NETWORK_PLUGIN_DIR = 4;
+}
+
+message RuntimeConfig {
+    required ConfigKey key = 1;
```

Comment:
The convention is to not use any `required` fields, which has been removed in proto3.